flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -808,14 +808,8 @@ static bool updateRcvrActivityCompare(uintptr_t rcvr_id, struct rcvr_activity_fs
 			case MANUALCONTROLSETTINGS_CHANNELGROUPS_PPM:
 				group = RECEIVERACTIVITY_ACTIVEGROUP_PPM;
 				break;
-			case MANUALCONTROLSETTINGS_CHANNELGROUPS_DSMMAINPORT:
-				group = RECEIVERACTIVITY_ACTIVEGROUP_DSMMAINPORT;
-				break;
-			case MANUALCONTROLSETTINGS_CHANNELGROUPS_DSMFLEXIPORT:
-				group = RECEIVERACTIVITY_ACTIVEGROUP_DSMFLEXIPORT;
-				break;
-			case MANUALCONTROLSETTINGS_CHANNELGROUPS_DSMRCVRPORT:
-				group = RECEIVERACTIVITY_ACTIVEGROUP_DSMRCVRPORT;
+			case MANUALCONTROLSETTINGS_CHANNELGROUPS_DSM:
+				group = RECEIVERACTIVITY_ACTIVEGROUP_DSM;
```

Comment:
Why just DSM though?  It's a total hack specific to one receiver type. 

We could transition to have the channelgroup always specify the port instead.  Would let you duplicate any of these and would use less memory.  Probably not good for this PR though.